scripts/README.md

```diff
@@ -105,3 +105,8 @@ Mock pricer deployed! 🎉
 Transaction hash: 0xebf12fc6603ee857bcabe1c0c6194XX82639a9888776f7ff6e83a409XXXXXXXX
 Deployed contract address: 0x669cC97687c792fc5369d7bdd38cXXXXXXXXXX
 ```
+
+## Verification of Contracts 
+```sh
+truffle run verify ChainlinkPricer@0x17FDF6cE88517bf0Cff19e014E509B543DD78432  --network mainnet
+```
```

Comment:
Why we are adding this to README ?

Reply:
because I imagine whoever wants to deploy also needs to verify right? Would you add it to packag.json instead? :)